odoo/tools/image.py

```diff
@@ -422,12 +422,15 @@ def image_fix_orientation(image):
     """
     getexif = getattr(image, 'getexif', None) or getattr(image, '_getexif', None)  # support PIL < 6.0
     if getexif:
-        exif = getexif()
-        if exif:
-            orientation = exif.get(EXIF_TAG_ORIENTATION, 0)
-            for method in EXIF_TAG_ORIENTATION_TO_TRANSPOSE_METHODS.get(orientation, []):
-                image = image.transpose(method)
-            return image
+        try:
+            exif = getexif()
+            if exif:
+                orientation = exif.get(EXIF_TAG_ORIENTATION, 0)
+                for method in EXIF_TAG_ORIENTATION_TO_TRANSPOSE_METHODS.get(orientation, []):
+                    image = image.transpose(method)
+        except Exception:
```

Comment:
handle onnly `SyntaxError`
also update the pot file of this

Reply:
Requested changes are done.
and I used same error message, so no need to change pot file.
https://github.com/odoo/odoo/blob/6419a258d617ba01968ccbf2db186596a2dc2932/odoo/addons/base/i18n/base.pot#L25868-L25869